rawlink-eio.opam

```diff
@@ -1,25 +1,8 @@
+# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
-authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
-license: "ISC"
-homepage: "https://github.com/haesbaert/rawlink"
-bug-reports: "https://github.com/haesbaert/rawlink/issues"
-dev-repo: "git+https://github.com/haesbaert/rawlink.git"
-doc: "https://haesbaert.github.io/rawlink/"
-build: [
-  [ "dune" "subst" ] {dev}
-  [ "dune" "build" "-p" name "-j" jobs ] ]
-depends: [
-  "ocaml" {>= "5.0.0"}
-  "dune" {>= "3.2"}
-  "rawlink" {>= "2.1"}
-  "eio" {>= "0.4"}
-]
-depexts: [
-  ["linux-headers"] {os-distribution = "alpine"}
```

Comment:
We lose this in the generated opam, is it possible to add this in dune so it gets generated as well ?

(same for the other opam files)